pyproject.toml

```diff
@@ -29,7 +29,7 @@ classifiers = [
     "Topic :: Utilities",
     "Typing :: Typed",
 ]
-dependencies = ["aiohttp", "pytz", "grapheme", "emoji"]
+dependencies = ["aiohttp", "pytz", "grapheme", "emoji", "requests"]
```

Comment:
With the use of the async webhook, we also don't need to add the dependency of requests